common/views/components/CookieNotice/CookieNotice.tsx

```diff
@@ -71,10 +79,14 @@ const CookieNotice: FunctionComponent = () => {
                 Read our policy
               </a>
             </div>
-            <CloseCookieNotice onClick={hideCookieNotice}>
-              <Icon icon={clear} />
-              <span className="visually-hidden">Close cookie notification</span>
-            </CloseCookieNotice>
+            {isEnhanced && (
+              <CloseCookieNotice onClick={hideCookieNotice}>
+                <Icon icon={clear} />
+                <span className="visually-hidden">
```

Comment:
Do we need to show this to people who don't have JS? Rephrasing as they'll have the same answer - do people without JS get cookies set?

Reply:
I had the same question, but @gestchild  pointed out that we do use cookies irrespective of whether you have JS. So I think we do have to show it.

Reply:
Two ways I can see this working then:
* We show it permanently for people without JS, making sure it doesn't [overlap with the UI](https://github.com/wellcomecollection/wellcomecollection.org/issues/6185)
* We have it link to an endpoint like `/cookies-accepted` that sets a cookie, `toggle_cookiesAccepted`, server side and we render dependant on that.

For this PR I would probaly remove the `isEnhanced` change and just have the tracking to finish that issue off.